app/views/coronavirus_landing_page/components/landing_page/_page_header.html.erb

```diff
@@ -23,31 +23,16 @@
       <div class="govuk-grid-row">
         <div class="govuk-grid-column-two-thirds">
           <%= render "govuk_publishing_components/components/heading", {
-            text: sanitize(header["title"]),
-            font_size: 19,
+            text: t("coronavirus_landing_page.header_title"),
+            font_size: 24,
             margin_bottom: 6,
             inverse: true
           } %>
-          <ul class="covid__instruction-list">
-            <% icons = [
-              "icon-hand-wash",
-              "icon-face-cover",
-              "icon-make-space"
-            ] %>
-            <% header["list"].each_with_index do | item, index | %>
-              <li class="covid__instruction-list-item">
-                <% if item.is_a?(Hash) %>
-                  <%= render "coronavirus_landing_page/components/landing_page/#{icons[index]}.svg" if icons[index] %>
-                  <span class="covid__instruction-text">
-                    <strong class="covid__instruction"><%= item["instruction"].capitalize %></strong>
-                    <%= item["detail"] %>
-                  </span>
-                <% else %>
-                  <%= item %>
-                <% end %>
-              </li>
-            <% end %>
-          </ul>
+
+          <%= render "govuk_publishing_components/components/govspeak", {
```

Comment:
We shouldn't need this as `render_govspeak` will spit out the component as per: https://github.com/alphagov/collections/blob/aa0ea47e99e3da23d631ec1badfab182a6038748/app/helpers/application_helper.rb#L56

Reply:
Indeed